prototype/src/main/webapp/view-meeting.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Meeting</title>
+    <script src="script.js"></script>
```

Comment:
I know this is a prototype, but surely there's a better name than `script.js`

Reply:
Renamed it to viewMeeting.js 😅